docs/index.html

```diff
@@ -1135,6 +1135,34 @@ <h2 id="inert-heading">works with inert elements</h2>
       </p>
     </div>
 
+
+    <div id="demo-dom-remove">
+      <h2 id="dom-remove-heading">Pulls focus when removing DOM elements</h2>
+      <p>
+      </p>
```

Comment:
I don't think the title really explains it well. How about adding this explanation here: "Keeps focus in the trap when the focused element is removed from the DOM."

Reply:
Agreed. I tried to give it another go in [46bf5cb](https://github.com/focus-trap/focus-trap/pull/932/commits/46bf5cb5c7309b8abd078a9a5da596f28393e73a).